### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by thomasritz**
This pull request fixes a couple of failing tests and removes some warnings.

Furthermore, it removes Ruby 1.9.2 from the Travis test suite.
And it completes the test matrix to test both Ruby 2.0.0 and 2.2.2 with ActiveResource 4.0, 4.1 and 4.x.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,16 +4,11 @@ source 'https://rubygems.org'
...
 gem "test-unit"
 gem "webmock"
 gem "pry"
 
 gem 'helpful_configuration'
-
-if RUBY_VERSION =~ /^1\.9/
-  gem 'json'
-  gem "redcarpet"
-  gem "yard"
```
Comment (apepper) [inline Gemfile]: Does yard documentation still works? Currently there is http://www.rubydoc.info/gems/infopark_crm_connector/1.2.0 .

File: Gemfile [hunk 2]
```diff
@@ -4,16 +4,11 @@ source 'https://rubygems.org'
 gemspec
 
 # Dependencies for test environment
+gem "appraisal"
 gem "rake"
 gem "rest-client"
 gem "test-unit"
 gem "webmock"
```
Comment (apepper) [inline Gemfile]: Please limit the version of `webmock`.

**Comment by apepper** [comment]
FEEDBACK (098a724)

Thank you for your contribution.

Here are some issues with this PR:
* [ ] Please ensure, that `yard` is still working (see details at https://github.com/infopark/crm_connector/pull/19#discussion-diff-78544045 ).
* [ ] Please limit the version of `webmock`, e.g. `~> 2.1` (see details at https://github.com/infopark/crm_connector/pull/19#discussion-diff-78544174).

File: Gemfile (hunk 2 above)
Comment (thomasritz) [inline Gemfile]: I haven't introduced `webmock`. Please add the version you would like.

File: Gemfile (hunk 1 above)
Comment (thomasritz) [inline Gemfile]: I don't know. I removed Ruby 1.9 from the tests. If Ruby 1.9 is still required for some tasks, it should be documented so. I'm going to revert those lines if it's required for yard…

Comment (apepper) [inline Gemfile]: `yard` and `redcarpet` are probably required. I don't know, why there are only in the ruby 1.9 block. Must be a bug.

**Comment by thomasritz** [comment]
@apepper I limited the webmock version to 2.1., I reverted the yard gemfile lines and added a yard task to the travis build (build it with Ruby 2.0.0). Travis is green. Since RubyDoc.info is powered by yard 0.9.5 (ruby-2.3.1) I hope yard doc works again.

**Comment by apepper** [comment]
OK (221e898)

Great job 🎉

**Pull request merged**